serdect/README.md

```diff
@@ -29,6 +29,18 @@ While this crate can't ensure that format implementations don't perform
 other kinds of data-dependent branching on the contents of the serialized data,
 using a constant-time hex serialization with human-readable formats should
 help reduce the overall timing variability.
+The table below lists the formats `serdect` is tested against, and
+whether the serialization via `array` or `slice` is constant-time for them.
```

Comment:
I think we should be careful when using the word "constant-time", as pointed out above, this is best effort, we can't actually guarantee it.
```suggestion
The table below lists the formats `serdect` is tested against, and
whether the serialization via `array` or `slice` is data-independent.
```

Reply:
That whole paragraph is largely a placeholder, if someone can suggest a better wording for it, I will happily accept.

Reply:
Rephrased.